src/movie/movie.ts

```diff
@@ -273,6 +273,8 @@ export class Movie {
    * @param [options.type='video/webm'] - MIME type for exported video
    * @param [options.onStart] - Called when the recording starts
    * @return Resolves when done recording, rejects when media recorder errors
+   * @deprecated Use individual options instead of mediaRecorderOptions.
```

Comment:
Can you add an example of which individual options we're referring to (for users)? Something like

```ts
Use individual options (such as `type`) instead.
```